src/index.html

```diff
@@ -26,53 +26,41 @@ <h3>Score:<span id='player-2-score'> 0 </span></h3>
           <article class='player-scorecard player3 3'>
             <h2 id='js-player3-name'>Player3</h2>
             <h3>Score:<span id='player-3-score'> 0 </span></h3>
+          <button id="submit-button">Submit Guess</button>
           </article>
         </section>
       <section id="puzzle-table">
         <div class="grid">
-          <nav class="grid-item"> 
-              <h1>Activity Tracker</h1>
-              <h2 id="user-name__display"> ... </h2>
-              <div id="current-date__display">
-                Current Date
-              </div>
-              <ul>
-                <li id="user-full-name__display"></li>
-                <li id="user-address__display">H2O</li>
-                <li id="user-email__display">Sleep</li>
-                <li id="user-daily-step-goal__display">Steps</li>
-              </ul>
-            </nav>
           <table id="tg">
               <tr>
-                <th class="tg-0lax"></th>
-                <th class="tg-0lax"></th>
-                <th class="tg-0lax"></th>
-                <th class="tg-0lax"></th>
+                <th id="column0-row0__category" class="tg-0lax">cat0</th>
```

Comment:
Interested in your thoughts on the class naming conventions for our classes.

Reply:
This is fine and sometimes a common convention when you have rows/columns to number them as you did. I think it’s pretty clear what they’e referring to so I’d leave as-is.